ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/host_vars/host3/snmp-settings-avd-6-behavior.yml

```diff
@@ -9,13 +9,6 @@ l2leaf: # dynamic_key: node_type
     # Testing uplink_structured_config on l2leaf
     uplink_switches: [host1]
     uplink_switch_interfaces: [Ethernet4]
```

Comment:
you can remove these lines as well - the purpose of the uplink was only to test the uplink_structured_config